src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,182 @@
+import clsx from 'clsx';
+import React, {
+  useId,
+  type ChangeEventHandler,
+  type CSSProperties,
+} from 'react';
+import Label from '../Label';
+import Text from '../Text';
+import styles from './Slider.module.css';
+
+/**
+ * Returns the lowest multiple of 10 that multiplies with all numbers in a list to make them integers.
+ * Useful for floating point math.
+ * @example
+ * lowestTenMultiplier([-2.212, 0.1, 2.0, 100, 1000.01])
+ * // returns 1000
+ * @param numbers
+ * @returns {number} Lowest multiple of 10.
+ */
+const lowestTenMultiplier = (numbers: number[]): number => {
+  let multiplier = 1;
+  while (
+    numbers.some((number) => !Number.isInteger((number * multiplier) % 1))
+  ) {
+    multiplier *= 10;
+  }
+  return multiplier;
+};
```

Comment:
nit: we can also save this to src/utils and write separate unit tests for it, since this is a generic piece of functionality.  The tests for this would serve as a good set of examples too.

Reply:
Moved to `src/util/` and wrote some tests 👍